pkg/venafi/tpp/connector_test.go

```diff
@@ -51,29 +51,7 @@ var ctx *test.Context
 
 func init() {
 	ctx = test.GetEnvContext()
-	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{
```

Comment:
Why did you remove this code?

Reply:
Oops. That is my mistake, I will re-add this bit.